### PR TITLE
Email capture

### PR DESCRIPTION
**Pull request by davidbielik**
* Email capture and thank you page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,3 +59,5 @@ To view our legacy templates which used `.zip` files, [click here][5].
 [101]: braze-templates/1-braze-dashboard-carousel-modal
 [102]: braze-templates/2-braze-dashboard-simple-modal
 [103]: braze-templates/3-braze-dashboard-survey-modal
+[104]: braze-templates/4-sms-capture-modal
```
Comment (davidbielik) [inline README.md:62]:
```suggestion
[104]: braze-templates/4-sms-capture
```

File: README.md [hunk 2]
```diff
@@ -58,3 +59,5 @@ To view our legacy templates which used `.zip` files, [click here][5].
...
+[105]: braze-templates/5-email-capture-modal
```
Comment (davidbielik) [inline README.md:63]:
```suggestion
[105]: braze-templates/5-email-capture
```

File: README.md [hunk 3]
```diff
@@ -58,3 +59,5 @@ To view our legacy templates which used `.zip` files, [click here][5].
 [101]: braze-templates/1-braze-dashboard-carousel-modal
 [102]: braze-templates/2-braze-dashboard-simple-modal
 [103]: braze-templates/3-braze-dashboard-survey-modal
+[104]: braze-templates/4-sms-capture
```
Comment (davidbielik) [inline README.md]:
```suggestion
[104]: braze-templates/4-sms-capture-modal
```

File: README.md [hunk 4]
```diff
@@ -58,3 +59,5 @@ To view our legacy templates which used `.zip` files, [click here][5].
...
+[105]: braze-templates/5-email-capture
```
Comment (davidbielik) [inline README.md]:
```suggestion
[105]: braze-templates/5-email-capture-modal
```

File: braze-templates/5-email-capture-modal/index.html [hunk 5]
```diff
@@ -0,0 +1,622 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Single Page Modal</title>
```
Comment (davidbielik) [inline braze-templates/5-email-capture-modal/index.html]:
```suggestion
    <title>Email Signup</title>
```

File: braze-templates/5-email-capture-modal/index.html [hunk 6]
```diff
@@ -0,0 +1,621 @@
...
+                        <path fill-rule="evenodd" clip-rule="evenodd"
+                            d="M14.4175 16.8419L8.51467 10.9391C7.8452 10.2696 7.8452 9.18422 8.51467 8.51475C9.18414 7.84528 10.2696 7.84528 10.939 8.51475L16.8418 14.4176L22.7446 8.51475C23.4141 7.84528 24.4995 7.84528 25.169 8.51475C25.8385 9.18422 25.8385 10.2696 25.169 10.9391L19.2662 16.8419L25.169 22.7447C25.8385 23.4142 25.8385 24.4996 25.169 25.1691C24.4995 25.8386 23.4141 25.8386 22.7446 25.1691L16.8418 19.2663L10.939 25.1691C10.2696 25.8386 9.18414 25.8386 8.51467 25.1691C7.8452 24.4996 7.8452 23.4142 8.51467 22.7447L14.4175 16.8419Z"
+                            fill="#A8B3B8" />
+                    </svg>
+                </button>
+                <h1 class="modal__title">Thanks for signing up!</h1>
+                <!-- <h2 class="modal__sub-title">Sub head maybe here that is not too long</h2> -->
+                <p class="modal__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.</p>
+                <div class="modal__button-wrap">
+                    <a class="modal__btn modal__btn--main" data-button-id="learn more button">Learn
```
Comment (wesleyorbin) [inline braze-templates/5-email-capture-modal/index.html]: Maybe add some instructions here to either add an `href` or remove this button? I could see it being easily forgotten.

```suggestion
                    <a class="modal__btn modal__btn--main" data-button-id="learn-more">Learn
```

**Review by wesleyorbin — APPROVED** [review]

**Pull request merged**